app/views/movies/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1><%= @movie.title %></h1>
+<p>Creation Year: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+
+<br>
+
+<%= form_with(model: @movie, url: url_for(controller: 'movies', action: 'add_actor', id: @movie), method: :post, local: true, turbo: :false) do |form| %>
+  <%= form.label :actor_id, 'Add Actor by ID' %>
+  <%= form.text_field :actor_id %>
+  <%= form.submit 'Add Actor' %>
+<% end %>
```

Comment:
Yea this form is unfortunately incorrect in it's implementation. Currently your form is posting to `/movies/:id/add_actor` when it's suppose to be posting to `/movies/:id/actors` since we are creating an association between an actor and a movie.

The updated syntax should look something like this: `<%= form_with url: "/movies/#{@movie.id}/actors, method: :post, local: true, data: { turbo: false } do |form| %>`

In your routes the verb+path would be: `post "/movies/:id/actors", to: "actor_movies#create"`, assuming you have an `ActorMoviesController`, else it would go to `movies#create`. But again, I talked about why it's easier to go to a joins controller before.